cmd/openshift-install/create.go

```diff
@@ -77,13 +77,35 @@ var (
 		name: "Ignition Configs",
 		command: &cobra.Command{
 			Use:   "ignition-configs",
-			Short: "Generates the Ignition Config asset",
+			Short: "(deprecated) Generates the Ignition Config asset",
 			// FIXME: add longer descriptions for our commands with examples for better UX.
 			// Long:  "",
+			PreRun: func(_ *cobra.Command, _ []string) {
+				logrus.Warning("The ignition-configs sub-command has been deprecated. Use the node-config or pre-cluster sub-commands instead.")
+			},
 		},
 		assets: []asset.WritableAsset{&bootstrap.Bootstrap{}, &machine.Master{}, &machine.Worker{}, &kubeconfig.Admin{}, &cluster.Metadata{}},
 	}
 
+	nodeConfigTarget = target{
+		name: "Node Config",
+		command: &cobra.Command{
+			Use:   "node-config",
+			Short: "Generates the ignition configs for the master and worker nodes",
+		},
+		assets: []asset.WritableAsset{&machine.Master{}, &machine.Worker{}},
```

Comment:
Actually, who cares about these pointer Ignition configs?  Do we even need a target for them?  I expect the default pointers will be fine and folks will tweak the machine Ignition configuration by configuring the machine-config server.  `pre-cluster` may be sufficient with it's reduced target set.

Reply:
I don't know. I thought that @crawford had a use case for modifying the master and worker ignition configs.